packages/cli/src/services/orchestration/main/handle-worker-response-message-main.ts

```diff
@@ -27,7 +29,7 @@ export async function handleWorkerResponseMessageMain(
 		case 'getStatus':
 			Container.get(Push).broadcast('sendWorkerStatusMessage', {
 				workerId: workerResponse.workerId,
-				status: workerResponse.payload,
+				status: workerResponse.payload as WorkerStatus,
```

Comment:
Could we get rid of this cast? Is `payload` correctly typed?

Reply:
I tried, but it turned out to be complicated. I think we need to refactor the redis commands a bit in another PR.